test/cmd/apply.sh

```diff
@@ -165,7 +165,7 @@ __EOF__
 
   # same thing without prune for a sanity check
   # Pre-Condition: no POD exists
-  kube::test::get_object_assert pods "{{range.items}}{{${id_field:?}}}:{{end}}" ''
+  kube::test::wait_object_assert pods "{{range.items}}{{${id_field:?}}}:{{end}}" ''
```

Comment:
`kubectl delete pods` waits for pods to actually be deleted by default before returning... did you actually see flakes here?

Reply:
I was overzealous here too.